staging/README.md

```diff
@@ -87,7 +87,7 @@ import (
   ./hack/update-vendor.sh
 ```
 
-8. Run [`./hack/update-go-workspace.sh`](/hack/update-go-workspace.sh) to add
+8. Run [`./hack/update-vendor.sh`](/hack/update-vendor.sh) to add
```

Comment:
This script is already referenced in step 7, combine 7 and 8

Reply:
Oops! didn't remember that I already ran it by the time I got to updating. 

Have removed step 8.